framework/src/org/checkerframework/framework/qual/PostconditionAnnotation.java

```diff
@@ -16,6 +16,12 @@
  * <p>The value {@code qualifier} that is necessary for a postcondition specified with {@link
  * EnsuresQualifier} is hard-coded here with the value {@code qualifier}.
  *
+ * <p>Additionally, the elements of the postcondition annotation (annotated by this meta-annotation)
+ * can be used to specify values of arguments of the qualifier. Each such element must be annotated
+ * by {@link QualifierArgument}, with a value specifying the name of the target qualifier argument.
+ * If no value is not specified, the name of the element is used. The element must have the same
```

Comment:
"no value is not specified" looks like a typo.  Same in PreconditionAnnotation.java